prometheus/counter.go

```diff
@@ -35,6 +35,9 @@ type Counter interface {
 	// Prometheus metric. Do not use it for regular handling of a
 	// Prometheus counter (as it can be used to break the contract of
 	// monotonically increasing values).
+	// This method is DEPRECATED. Use NewConstMetric to create a counter for
```

Comment:
Some background on that convention https://groups.google.com/forum/#!topic/golang-dev/ya5vE0G62Z4